models/articles.models.js

```diff
@@ -75,4 +75,21 @@ exports.insertCommentByArticleId = (article_id, comment) => {
     .then((result) => {
         return result.rows[0];
     });
-}
\ No newline at end of file
+}
+
+exports.updateArticleById = (article_id, articleUpdate) => {
+    const query =
+        `UPDATE articles
+        SET votes = votes + $1
+        WHERE article_id = $2
+        RETURNING *`;
+
+    return db.query(query, [articleUpdate.inc_votes, article_id])
+    .then((result) => {
+        if(result.rows.length !== 0) {
```

Comment:
We need to check whether the article exists or not in order to determined the reason a patch may fail.If it does not exist, it should throw a 404. There could be multiple reasons we get back an empty row.